docs/user_guide/model-registration.rst

```diff
@@ -5,84 +5,60 @@
 Model registration
 ==================
 
-This guide presents a quick instruction for "registering a model" in a project workflow.
+This guide presents instructions for "registering a model" for a project.
 
-It is still recommended to read the detailed instructions under :ref:`model_mapping` and
-:ref:`region` in particular.
-
-Region processing
------------------
-
-"Registering a model" for a **nomenclature**-based project workflow requires two
-specifications: 
+Model registration for a **nomenclature**-based project requires two specifications:
 
+* | a list of region names as they should appear in the processed scenario data
+  | (e.g., after processing as part of the upload to a *Scenario Explorer* instance)
 * a model mapping to perform region aggregation from *native_regions* to
   *common_regions* and renaming of model native regions (optional)
-* a list of region names as they should appear in the processed scenario data
-
-Model mapping
-^^^^^^^^^^^^^
 
-For this guide we will consider a model mapping as an example:
+Please read the detailed explanations of :ref:`region` and :ref:`model_mapping` before
+proceeding with model registration.
 
-.. code-block:: yaml
+Option 1) Registration using an Excel template
+----------------------------------------------
 
-    model: model_a
-    native_regions:
-      - region_1: model_a|Region 1
-      - region_2: model_a|Region 2
-      - region_3: model_a|Region 3
-    common_regions:
-      - World:
-        - region_1
-        - region_2
-        - region_3
-      - Common region 1:
-        - region_1
-        - region_2
+Please use the `Excel template`_ and send it to the project managers by email.
 
-``model_a`` reports three regions natively, ``region_1``, ``region_2`` and ``region_3``.
-These three are to be renamed to the ``{model}|{region}`` pattern as is common place in
-many projects. For the ``common_regions``, there are ``World`` and ``Common region 1``.
+.. _`Excel template`: https://raw.githubusercontent.com/IAMconsortium/nomenclature/main/templates/model-registration-template.xlsx
 
-*For more details on the model-native and common regions refer to*
-:ref:`native-vs-common-region`.
+Option 2) Registration using a GitHub pull request
```

Comment:
I would flip this around and make this Option 1 and write as the second sentence something along the lines of:
"If you are not comfortable or familiar with GitHub pull requests you can also register your model using an Excel template. You can find the instructions for that here (link)".
Otherwise I fear that people, even the ones who could open a PR, just stop reading after option 1 and use the Excel template.

Reply:
I disagree - the GitHub-section is more than one page long, so users would not immediately see it. The situation where someone sends us the xlsx file (who could do a PR) is less worrisome compared to the situation where a non-expert users wastes an hour trying to create a PR (and we then have to clean up the mess).